app/controllers/follow_ups_controller.rb

```diff
@@ -2,8 +2,14 @@
 
 class FollowUpsController < ApplicationController
   def create
-    f = FollowUp.new(data_attrs[:email], data_attrs[:language_id], data_attrs[:destination_id], data_attrs[:name])
+    f = FollowUp.create!(data_attrs.permit(permitted_params))
```

Comment:
Move the `data_attrs.permit()` call down to the permitted params method so that this line becomes 

```ruby
f = FollowUp.create!(permitted_params)
```